app/server/lib/Authorizer.ts

```diff
@@ -358,6 +359,10 @@ export async function addRequestUser(
           mreq.user = user;
           mreq.userId = user.id;
           mreq.userIsAuthorized = true;
+          const today = moment().startOf('day');
+          if (today !== moment(user.lastConnectionAt).startOf('day')) {
+            await dbManager.updateUser(mreq.userId, {lastConnectionAt: today.toDate()});
```

Comment:
Could we do this from `getUserByLoginWithRetry` instead? It also (optionally) updates the user.